css/src/editor.scss

```diff
@@ -413,6 +418,10 @@ div[class^="wp-block-block-lab-"] {
     }
   }
 
+  .block-lab-color-control .components-base-control__label {
+    display: block;
```

Comment:
This is a similar fix to [this](https://github.com/getblocklab/block-lab/pull/467#discussion_r341829224), but for the Color field in the Inspector.